modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/AddFormInstanceRecordMVCResourceCommand.java

```diff
@@ -160,6 +177,13 @@ protected DDMForm getDDMForm(DDMFormInstance ddmFormInstance)
 		return ddmStructure.getDDMForm();
 	}
 
+	private static final Log _log = LogFactoryUtil.getLog(
+		AddFormInstanceRecordMVCResourceCommand.class);
+
+	@Reference
+	private AddFormInstanceRecordMVCCommandHelper
+		_addFormInstanceMVCCommandHelper;
```

Comment:
@rodrigopaulino shouldn’t we named it as `_addFormInstanceRecordMVCCommandHelper`?

Reply:
Totally. My bad!